hw/ip_templates/pinmux/syn/pinmux_syn_cfg.hjson

```diff
@@ -6,20 +6,13 @@
   name: pinmux
 
   // Fusesoc core file used for building the file list.
-  fusesoc_core: lowrisc:ip:pinmux:0.1
+  fusesoc_core: lowrisc:ip_interfaces:pinmux:0.1
 
   import_cfgs: [// Project wide common synthesis config file
                 "{proj_root}/hw/syn/tools/dvsim/common_syn_cfg.hjson"],
 
-  overrides: [
-    {
-      name: design_level
-      value: "top"
-    }
-  ]
-
   // Timing constraints for this module
-  sdc_file: "{proj_root}/hw/ip/pinmux/syn/constraints.sdc"
+  sdc_file: "{proj_root}/hw/top_earlgrey/ip_autogen/pinmux/syn/constraints.sdc"
```

Comment:
Is there a way to make this path use the name of the top instead of hardcoding earlgrey?

Reply:
Good point. I will make this a template file.